ci/script/run_integrated_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+wget https://storage.googleapis.com/dart-archive/channels/stable/release/latest/sdk/dartsdk-linux-x64-release.zip
+unzip dartsdk-linux-x64-release.zip
+CURRENT_DIR="$(pwd)"
+
+git clone https://github.com/IOT-DSA/dslink-dart-test -b repacking-tested-links
```

Comment:
The `-b` needs to be removed when https://github.com/IOT-DSA/dslink-dart-test/pull/10 gets merged. So don't merge this PR yet.